sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
Adding one that checks if ``sample_weights`` is equivalent to doing the corresponding ``class_weights`` would be nice, too. I feel it should be pretty straight-forward, though. Why would you need a helper? Just check if ``decision_function`` is ``almost_equal``.